### DISCUSSION TITLE
What is the easiest way to retrieve data from ceph backup

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#8480] [Q&A]
Hello, we made a backup of the whole cephcluster with all of its resources. If we want to restore a directory from the Ceph Filesystem we had to restore the whole cluster than rsync that directory to the original ceph filesystem, is there any simpler way to restore these files from the OSD PVS without the need to restore the whole cluster ?

**Comment by travisn** [comment]
What did you use to backup and restore? If you use a tool such as Velero it can backup individual PVs.

**Reply by ahmedTouati** [reply]
@travisn yes I use Velero but by backing up and restoring only PVs can I access data in the filesystem without restoring the whole cephcluster ?

**Reply by ahmedTouati** [reply]
> What did you use to backup and restore? If you use a tool such as Velero it can backup individual PVs.

Is that possible ?

**Reply by travisn** [reply]
I understood Velero would backup the contents of individual PVs and would not require restoring the entire Ceph cluster, but I have not tried it.
